python/import_util.py

```diff
@@ -0,0 +1,93 @@
+from mage.export_import_util.parameters import Parameter
+import mgp
+import json as js
+
+
+def create_vertex(ctx, properties, labels):
+    vertex = ctx.graph.create_vertex()
+    vertex_properties = vertex.properties
+
+    for key, value in properties.items():
+        vertex_properties[key] = value
+
+    for label in labels:
+        vertex.add_label(label)
+
+    return vertex.id
+
+
+def create_edge(ctx, properties, start_node_id, end_node_id, type, vertex_ids):
+    vertex_from = ctx.graph.get_vertex_by_id(vertex_ids[start_node_id])
+    vertex_to = ctx.graph.get_vertex_by_id(vertex_ids[end_node_id])
+    edge = ctx.graph.create_edge(vertex_from, vertex_to, mgp.EdgeType(type))
+    edge_properties = edge.properties
+
+    for key, value in properties.items():
+        edge_properties[key] = value
+
+
+@mgp.write_proc
+def json(ctx: mgp.ProcCtx, path: str) -> mgp.Record():
+    """
+    Procedure to import the local JSON created by the export_util.json procedure.
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being imported.
+    """
+    file = open(path)
```

Comment:
Use try/catch and use ` with` statement for opening, because file handler might remain open